frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -420,6 +448,12 @@ const DisabledPopover: React.FC<DisabledPopoverProps> = ({ operatorHub, sourceNa
   );
 };
 
+const getRegistryPollInterval = (catalogSource: CatalogSourceKind): string => {
+  return catalogSource.spec?.updateStrategy
+    ? catalogSource.spec.updateStrategy.registryPoll.interval
```

Comment:
Let's handle the case where `registryPoll` is missing

```suggestion
    ? catalogSource.spec.updateStrategy.registryPoll?.interval
```